lib/main.dart

```diff
@@ -9,20 +15,111 @@ class MyApp extends StatelessWidget {
   Widget build(BuildContext context) {
     return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
-        // This is the theme of your application.
-        //
-        // Try running your application with "flutter run". You'll see the
-        // application has a blue toolbar. Then, without quitting the app, try
-        // changing the primarySwatch below to Colors.green and then invoke
-        // "hot reload" (press "r" in the console where you ran "flutter run",
-        // or simply save your changes to "hot reload" in a Flutter IDE).
-        // Notice that the counter didn't reset back to zero; the application
-        // is not restarted.
-        primarySwatch: Colors.blue,
+      title: 'Playlister',
+      theme: darkTheme,
+      home: const MyHomePage(title: 'Playlister'),
+    );
+  }
+}
+
+class RecentlyUpdatedListItem extends StatelessWidget {
+  final String text;
+  final Color color;
+
+  const RecentlyUpdatedListItem({
+    Key? key,
+    required this.text,
+    required this.color,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 100,
+      margin: const EdgeInsets.symmetric(
+        horizontal: 10,
+      ),
+      child: ElevatedButton(
+        style: ElevatedButton.styleFrom(
+          primary: color,
+          shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(
+              15,
+            ),
+            side: const BorderSide(
+              color: Colors.white,
+              width: 1.5,
+            ),
+          ),
+        ),
+        onPressed: () {},
+        child: Padding(
+          padding: const EdgeInsets.only(
+            right: 10,
+            bottom: 10,
+          ),
+          child: Align(
+            alignment: Alignment.bottomLeft,
+            child: Text(
+              text,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class RecentlyAddedListItem extends StatelessWidget {
+  final String text;
+  final Color color;
+
+  const RecentlyAddedListItem({
+    Key? key,
+    required this.text,
+    required this.color,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(
+        top: 5,
+        bottom: 5,
+      ),
+      child: SizedBox(
+        height: 45,
+        child: ElevatedButton(
+          style: ElevatedButton.styleFrom(
+            primary: Colors.grey[800],
+            elevation: 0,
+            shape: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(
+                15,
+              ),
+            ),
+          ),
+          onPressed: () {},
+          child: Row(
+            children: [
+              Container(
+                decoration: BoxDecoration(
+                  color: color,
+                  shape: BoxShape.circle,
+                ),
+                margin: const EdgeInsets.only(
```

Comment:
Put a `SizedBox(width: 10)` between the container and the text instead of using margin